R/user-record.R

```diff
@@ -0,0 +1,25 @@
+#' Return Crunch User Information
+#'
+#' This function returns information about the current user of the shiny app. This
+#' is useful if you want to change the behaviour of the app dependeing on who is
+#' viewing the app.
+#' 
+#' @param ... Additional arguments passed to `.buildReactiveExpr`
+#' @return A list with the following entries:
+#' - id_method: the method used to authenticate the user, for instance oauth
+#' - preferneces: various preferences used by the Crunch web app
```

Comment:
Spelling; also I'm not sure that an exhaustive list of the attributes is worth including. Some are a little embarrassing to document that our API looks like that; those attributes are also probably not relevant. "name" and "email" are probably the only ones worth noting.